docs/articles/documentation/recipes/create-a-data-driven-test-case.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: docs
+title: Create Data-Driven Tests
+permalink: /documentation/recipes/create-data-driven-tests.html
+---
+# Create Data-Driven Tests
+
+[Data-driven testing](https://en.wikipedia.org/wiki/Data-driven_testing) is a procedure when you repeat the same test scenario with different input parameters and then verify the result with the given output values.
+
+Assume you have a `data.json` file with data objects (*test cases*) that consist of two input values (`name` and `comment`) and a value that specifies the expected result (`resultText`).
+
+```json
+[
+    {
+        "name": "John Heart",
+        "comment": "I love TestCafe!",
+        "resultText": "Thank you, John Heart!"
+    },
+    {
+        "name": "Olivia Peyton",
+        "comment": "TestCafe is awesome!",
+        "resultText": "Thank you, Olivia Peyton!"
+    }
+]
+```
+
+To create data-driven tests, iterate through the test cases, call the [test](../test-api/test-code-structure.md#tests) method at each iteration and write [test actions](../test-api/actions/README.md) and [assertions](../test-api/assertions/README.md) that use test case parameters.
+
+```js
+import { Selector } from 'testcafe';
+
+const dataSet = require('./data.json');
```

Comment:
we can use `import` here too

Reply:
AFAIK, `import` doesn't parse JSON, does it?

Reply:
you're right, but import also works fine in ES6. I don't mind leaving your variant.